README.md

```diff
@@ -1,4 +1,4 @@
-# AngularTourOfHeroes
+# Created from AngularTourOfHeroes
```

Comment:
Doesn't seem relevant. This is just a repo to play around.

Reply:
Agreed, but I'd rather have an app-oriented title.